apps/tutorial_servo_control/src/main.cpp

```diff
@@ -0,0 +1,23 @@
+#include "mbed.h"
+#include "TutorialServo.h"
+#include "CANMsg.h"
+
+
+CAN can(CAN1_RX, CAN1_TX);
+
+int main() {
+
+        TutorialServo servo(PA_1);
+        CANMessage msg;
+
+        while (true) {
+                if(can.read(msg)){
+
+                        printf("Message received: %d\n", msg.data[0]);
+                        float range{servo.getRangeInDegrees()};
+                        servo.setPositionInDegrees(range * msg.data[0]);
```

Comment:
you dont need the range. The can message should carry a value between 0 and 1. u pass that into the function. inside the funciton, u should map that accordingly.